azkaban-common/build.gradle

```diff
@@ -25,24 +20,18 @@ model {
 
 dependencies {
   compile('com.google.guava:guava:13.0.1')
-  compile('com.h2database:h2:1.3.170')
-  compile('commons-codec:commons-codec:1.9')
   compile('commons-collections:commons-collections:3.2.2')
-  compile('commons-configuration:commons-configuration:1.8')
   compile('commons-dbcp:commons-dbcp:1.4')
   compile('commons-dbutils:commons-dbutils:1.5')
   compile('commons-fileupload:commons-fileupload:1.2.1')
   compile('commons-io:commons-io:2.4')
   compile('commons-lang:commons-lang:2.6')
-  compile('commons-logging:commons-logging:1.1.1')
-  compile('commons-pool:commons-pool:1.6')
   compile('javax.mail:mail:1.4.5')
   compile('javax.servlet:servlet-api:2.5')
```

Comment:
They need not be right? If 2 submodules are independent of each, there may not be relation between the version numbers that each of them use. And also we can upgrade each individually for that matter.

If however, the submodules are dependent, then the dependent submodule will automatically import the dependencies of the parent submodule since I turned transitive dependencies on. It was disabled previously.